### PR TITLE
CLOUD-2315 Add .GA to the end of all instances of the product version

### PR DESCRIPTION
**Pull request by ASzc**

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -2,22 +2,22 @@ schema_version: 1
 
 name: "redhat-sso-7/sso72"
 description: "Red Hat Single Sign-On 7.2 container image"
-version: "7.2.0"
+version: "7.2.0.GA"
```
Comment (goldmann) [inline image.yaml]: It should be left without `.GA` here.

**Comment by ASzc** [comment]
Pushed a replacement commit with `version: "7.2.0"` instead of `version: "7.2.0.GA"`

**Review by goldmann — APPROVED** [review]
LGTM

**Comment by ASzc** [comment]
Thanks!

**Pull request merged**